src/ol/control/ScaleLine.js

```diff
@@ -69,7 +76,8 @@ class ScaleLine extends Control {
 
     const options = opt_options ? opt_options : {};
 
-    const className = options.className !== undefined ? options.className : 'ol-scale-line';
+    const className = options.className !== undefined ? options.className :
+      options.bar !== undefined ? 'ol-scale-bar' : 'ol-scale-line';
```

Comment:
```suggestion
      options.bar ? 'ol-scale-bar' : 'ol-scale-line';
```

Without this change, `bar: false` would result in `ol-scale-bar` for a class name, which is not what we want.

Reply:
right, fixed